app/scripts/localisations/translation.js

```diff
@@ -1168,197 +1168,7 @@ define([], function () {
       simCard: 'SIM-Karte'
```

Comment:
Yes, I wanted to remove the whole file but get a error. I can't find where to remove the requirements to this file. That is why it is still there.... I can empty this file for now if you want.